qiita_pet/handlers/analysis_handlers.py

```diff
@@ -141,14 +142,17 @@ def get(self, analysis_id):
             jobres[jobject.datatype].append((jobject.command[0],
                                              jobject.results))
 
-        dropped = {}
         dropped_samples = analysis.dropped_samples
+        dropped = defaultdict(list)
         if dropped_samples:
-            for proc_data_id, samples in viewitems(dropped_samples):
+            for proc_data_id, samples in viewitems(analysis.dropped_samples):
```

Comment:
1. analysis.dropped_samples -> dropped_samples: will change, left behind by mistake
2. if statement is necessary as dropped_samples is either a dict or None.

Thanks!